components/webflow/actions/list-orders/list-orders.mjs

```diff
@@ -0,0 +1,39 @@
+import webflow from "../../webflow.app.mjs";
+
+export default {
+  key: "webflow-list-orders",
+  name: "List Orders",
+  description: "List orders",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    webflow,
+    siteId: {
+      propDefinition: [
+        webflow,
+        "sites",
+      ],
+    },
+    status: {
+      label: "Status",
+      description: "The status to filter the orders.",
+      type: "string",
+      options: [
+        "pending",
+        "refunded",
+        "dispute-lost",
+        "fulfilled",
+        "disputed",
+        "unfulfilled",
+      ],
```

Comment:
You can set a constants variable for these kind of options in properties. The suggestion is to create a file named `webflow/common/constants.mjs` and there export all your constants like
```suggestion
      options: constants.STATUSES,
```